R/opts.R

```diff
@@ -574,52 +497,14 @@ opt_table_lines <- function(data,
 #'
 #' @export
 opt_table_outline <- function(data,
-                              active = TRUE,
                               style = "solid",
                               width = px(3),
                               color = "#D3D3D3") {
 
   # Perform input object validation
   stop_if_not_gt(data = data)
```

Comment:
Consider if `style = "none"`, set `width` and `color` to `NULL` (and in the docs, make it clear that "if style is 'none', width and color are ignored")

Reply:
Implemented now.